Pax.cs

```diff
@@ -109,7 +109,8 @@ public class Frontend {
       Console.ResetColor();
 
       // FIXME accept a -j parameter to limit number of threads?
-      Parallel.ForEach(PaxConfig.deviceMap, device => device.Capture());
+      foreach (var device in PaxConfig.deviceMap)
+        device.StartCapture();
```

Comment:
From the commit message and the changed code alone, it's not obvious what's the error that's being fixed. It's useful to say specifically which error is being fixed in the commit message, and also how it's being fixed, both to help other developers understand the change, and also for posterity. In this case, the call to `Capture` has been replaced with a call to `StartCapture`. Was there anything in the documentation perhaps that suggested that one's more suitable than the other in this case?